DeploymentScripts/DeployWebsite.ps1

```diff
@@ -25,8 +25,11 @@ Function Deploy-Website($packageDirectory, $msDeployUrl, $msDeployUser, $msDeplo
                     "-setParam:name=`"KitosContext-Web.config Connection String`",value=`"{16}`" " + 
                     "-setParam:name=`"kitos_HangfireDB-Web.config Connection String`",value=`"{17}`" " +
                     "-setParam:name=`"DefaultUserPassword`",value=`"{18}`" " +
-                    "-setParam:name=`"UseDefaultPassword`",value=`"{19}`"") `
-    -f $msdeploy, $packageDirectory, $msDeployUrl, $msDeployUser, $msDeployPassword, $logLevel, $esUrl, $ssoGateway, $securityKeyString, $smtpFromMail, $smtpNwHost, $resetPwTtl, $baseUrl, $mailSuffix, $kitosEnvName, $buildNumber, $kitosDbConnectionString, $hangfireConnectionString, $defaultUserPassword, $useDefaultUserPassword)
+                    "-setParam:name=`"UseDefaultPassword`",value=`"{19}`" " +
+					"-setParam:name=`"SSOServiceProviderServer`",value=`"{20}`" " +
```

Comment:
Der er sket noget mystisk med din whitespace... står den til tabs i stedet for spaces?

Reply:
https://www.hanselman.com/blog/TabsVsSpacesAPeacefulResolutionWithEditorConfigInVisualStudioPlusNETExtensions.aspx

Reply:
Just fix it 😄